app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCaptureActivity.java

```diff
@@ -256,8 +232,8 @@ public void setProgramStage(String programStageUid) {
 
     @Override
     public void showRuleCalculation(Boolean shouldShow) {
-        if(binding.eventTabLayout.getSelectedTabPosition() == 0) {
-            binding.calculationIndicator.getRoot().setVisibility(shouldShow ? View.VISIBLE : View.GONE);
+        if (binding.eventTabLayout.getSelectedTabPosition() == 0) {
+//            binding.calculationIndicator.getRoot().setVisibility(shouldShow ? View.VISIBLE : View.GONE);
```

Comment:
???

Reply:
:S Don't know why i left it that way... will remove it